telemetry/definitions/commonDefinitions.json

```diff
@@ -312,6 +312,11 @@
             "type": "string",
             "description": "Language-specific identification. Examples: v4.6.1, netcoreapp3.1, nodejs12.x. Not AWS Lambda specific. Allows for additional details when other fields are opaque, such as the Lambda runtime value 'provided'."
         },
+        {
+            "name": "locale",
+            "type": "string",
+            "description": "Language-related user preference information. Examples: en-US, en-GB, etc."
```

Comment:
Any pros/cons to using allowedValues here?

Reply:
I avoided that since the allowedValues could go pretty large over time. Open to changing that if desired

Reply:
I agree with making this one open ended. That avoids a treadmill maintaining an enumeration